pkg/framework/plugins/podlifetime/pod_lifetime_test.go

```diff
@@ -337,6 +337,38 @@ func TestPodLifeTime(t *testing.T) {
 				}
 			},
 		},
+		{
```

Comment:
can you add tests for each status that you are adding

Reply:
Thanks for your review. 
Done. 
Please re-check it